slick.grid.js

```diff
@@ -1441,18 +1455,22 @@ if (typeof Slick === "undefined") {
     }
 
     function applyColumnWidths() {
-      var x = 0, w, rule, canvas;
+      var x = 0;
       for (var i = 0; i < columns.length; i++) {
-        w = columns[i].width;
-        rule = getColumnCssRules(i);
-        rule.left.style.left = x + "px";
-        canvas = i > options.pinnedColumn ? contentCanvas[1].width : contentCanvas[0].width;
-        rule.right.style.right = (canvas - x - w) + "px";
+        var column = columns[i];
+        var width = getColumnVisibleWidth(column);

```

Comment:
The best practice i've seen is to put all var declarations at top-of-scope (so how it was before).

Is there a good reason for doing it this way?